tests/pytorch/distributed/test_torch_fsdp2.py

```diff
@@ -36,7 +40,8 @@ def _run_test(fp_init, sharding_dims):
 @pytest.mark.skipif(not te.torch_version() >= (2, 4, 0), reason="Requires PyTorch 2.4.0+")
 @pytest.mark.parametrize("sharding_dims", ([NUM_PROCS], [2, NUM_PROCS // 2]))
 @pytest.mark.parametrize("fp8_init", (False, True))
-def test_distributed(fp8_init, sharding_dims):
+@pytest.mark.parametrize("recipe", ("delayed_scaling", "current_scaling", "mx_fp8_block_scaling"))
```

Comment:
We should skip recipe test cases if the recipe is not supported. See the logic in `distributed/test_numerics.py`: https://github.com/NVIDIA/TransformerEngine/blob/0e80c847845be58dbb88f46a0975786ddc823798/tests/pytorch/distributed/test_numerics.py#L55-L69